src/plugins/janus_videoroom.c

```diff
@@ -7729,6 +7794,45 @@ static json_t *janus_videoroom_process_synchronous_request(janus_videoroom_sessi
 			gboolean disabled = json_is_true(json_object_get(s, "disabled"));
 			/* Create a publisher stream */
 			ps = g_malloc0(sizeof(janus_videoroom_publisher_stream));
+			if(mtype == JANUS_VIDEOROOM_MEDIA_AUDIO || mtype == JANUS_VIDEOROOM_MEDIA_VIDEO) {
+				/* First of all, let's check if we need to setup an SRTP for remote publisher */
+				JANUS_LOG(LOG_HUGE, "trying to enable SRTP crypto (%s) suite (%d) for stream. \n", srtp_crypto, srtp_suite);
```

Comment:
I don't think the text in this log line is correct. You don't even know if SRTP is enabled, when you get here. I'd remove this log line entirely. But, in general:

s/trying/Trying

Unneeded dot and space before `\n`